tests.js

```diff
@@ -21,7 +20,7 @@ function parseLog(text, logServer) {
     // console.log(words);
     //test if is auction
     if (words[1] === 'AUCTIONS,') {
-        // client.streamAuction(text.replace('||', '|'), "GREEN");
+        fakeStreamAuction(text.replace('||', '|'), logServer); // for testing purposes, don't use client.streamAuction
```

Comment:
I'm actually curious why you needed to de-duplicate pipes, are they common in auction messages?

Reply:
Not very common but apparently || is discord markdown for spoiler or something like that.  There are other discord markdown I should probably be checking for too, but spoiler was really annoying for the "stream".

Reply:
Ahh, that makes sense! So just in case someone did this, like `/auc WTS FBSS 16k || Something Else 1k` it wouldn't do dumb stuff